torch_xla/csrc/tensor_impl.cpp

```diff
@@ -63,6 +63,8 @@ XLATensorImpl::XLATensorImpl(XLATensor&& tensor)
                       bridge::XlaDeviceToAtenDevice(tensor.GetDevice())),
       tensor_(c10::make_intrusive<XLATensor>(std::move(tensor))) {
   is_non_overlapping_and_dense_ = false;
+  const_cast<XLATensorImpl*>(this)->SetupSizeProperties();
+  set_sizes_and_strides(c10::SymIntArrayRef(sym_sizes_.data(), sym_sizes_.size()), c10::fromIntArrayRefSlow(strides_default()));
```

Comment:
what's this call for?

Reply:
It does `has_symbolic_sizes_strides_ = true;`:
https://github.com/pytorch/pytorch/blob/2c313e7b99b6070a3a7d640e4bc8bf2fe3acbcf1/c10/core/TensorImpl.cpp#L1152